Docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8.4-slim
+ENV PYTHONIOENCODING UTF-8
+ARG DEBIAN_FRONTEND=noninteractive
+
+COPY ./config /config
+
+ENV CONFIG_FILE_PATH=/config/config.py
```

Comment:
/work/gunicorn.conf.py would be the default location for this.

Reply:
5bc62af

If the user manually configures the gunicorn.conf.py file, adds gunicorn.conf.py to the docker container, and then enters the path to gunicorn.conf.py in the environment variable $CONFIG_FILE_PATH, the gunicorn runs based on the specified file.